src/components/atoms/OcSearchBar/OcSearchBar.vue

```diff
@@ -257,9 +257,8 @@ export default {
   &-input {
     @extend .uk-input;
 
-    background-color: var(--oc-color-input-bg);
-    border: 1px solid var(--oc-color-input-border);
-    border-radius: 3px;
+    border-radius: 25px;
+    border: 0px;
```

Comment:
```suggestion
    border: 0;
```
Should be sufficient. Normally you don't set a unit in css when the value is `0`.